spec/unit/create.spec.js

```diff
@@ -300,4 +301,23 @@ describe('create', function () {
             });
         });
     });
+
+    describe('writeNameForAndroidStudio', () => {
+        var project_path = path.join('some', 'path');
```

Comment:
We can reduce some code duplication by using the `beforeEach` test hook, which gets ran before each test.

```suggestion
        beforeEach(() => {
            spyOn(fs, 'ensureDirSync');
            spyOn(fs, 'writeFileSync');
        });
        var project_path = path.join('some', 'path');
```

Then in each individual test inside this describe block, those two methods are already spied on, so we can remove those `spyOn` lines in the individual tests.

Reply:
Glad to see the kind of review you made. You are just like me who can put more than 5 comments on the smallest PR where only a few lines are modified 😄 My team some times gets horrified because of this 😄 

Made the code DRY.